src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -19,7 +20,7 @@ public Movie add(Movie movie) {
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
-            session.save(movie);
+            session.persist(movie);
```

Comment:
Why did you change this and why do you use persist instead of save everywhere?

Reply:
save is deprecated, thus i use persist